tests/model/test_model.py

```diff
@@ -922,3 +923,32 @@ def test_handle_typing_event(self, mocker, model,
         model.handle_typing_event(event)
 
         assert model.controller.view.set_footer_text.called == called
+
+    @pytest.mark.parametrize('event, final_muted_streams, ', [
+        (
+            {'property': 'in_home_view',
+             'stream_id': 19,
+             'value': True},
+            {15}
+        ),
+        (
+            {'property': 'in_home_view',
+             'stream_id': 30,
+             'value': False},
+            {15, 19, 30}
+        )
+    ])
```

Comment:
minor: maybe add ids of `remove_19` and `add_30` ?

Reply:
Done :+1: